MangoTornado.md

```diff
@@ -0,0 +1,27 @@
+---
+name: "@MangoTornado"
+project: "Portable Multifunction Device"
+---
+
+# Portable Multifunction Device
+
+## Summary
+
+I'm planning on building a portable multifunction device, that can assist in general applications as well as penetration testing the smart/not-smart devices around my home. Previously I joined a cybersecurity competition and started getting into a lot of the digital aspects, however I recently found out about a tool that combines an entry level sdr, nfc, rfic, as well as access to GPIO pins called the Flipper Zero. I hope to use this device to expand my knowledge on cybersecurity in more physical applications as well as digital ones. I also want to start learning how to solder and create prototyping boards for testing air quality, measuring distances, sniffing packets, and etc.
+
+## Plan
+
+First I'm going to acquire a Flipper Zero and familiarize myself with its hardware and software development kits. Second, I'm going to buy a soldering iron and solder as well as prototyping boards so I can attach chips like the BME680 to the GPIO headers on the flipper, in order to monitor air quality easily on the go. For now I'm looking at air quality and measuring distance using the chip listed prior and the HC-SR04 for ultrasonic distance measuring. Third, I need to solder the chips onto the boards, and write software that can convert the chip's output data into readable human format and output it on the Flipper Zero screen
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Flipper Zero    | [https://shop.flipperzero.one/](https://shop.flipperzero.one/)         | $169.00  |
+| Adafruit BME688 | [https://www.adafruit.com/product/5046](https://www.adafruit.com/product/5046)  | $19.95 |
+| HC-SR04 | [https://www.adafruit.com/product/3942](https://www.adafruit.com/product/3942)  | $3.95 |
+| PINECIL – Smart Mini Portable Soldering Iron | [https://www.amazon.com/dp/B096X6SG13](https://www.amazon.com/dp/B096X6SG13)  | $39.99 |
+| Hakko 633-01 633 Soldering Iron Holder | [https://www.amazon.com/dp/B000TGNWCS](https://www.amazon.com/dp/B000TGNWCS)  | $22.20 |
+| Prototyping Boards for Flipper Zero | [https://shop.flipperzero.one/products/proto-boards](https://shop.flipperzero.one/products/proto-boards)  | $10.00 |
+| Dupont Wire | [https://www.amazon.com/dp/B01EV70C78/](https://www.amazon.com/dp/B01EV70C78/)  | $6.98 |
```

Comment:
Will you need any passives (resistors, capacitors, diodes, etc.) for your circuit?

Reply:
The ultrasonic chip HC-SR04 comes with 2 resistors, and I don't think any are necessary for the BME